src/components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div className="buttonPanel">
+    <div>
+      <Button value="AC" />
+      <Button value="+/-" />
+      <Button value="%" />
+      <Button value="&divide;" />
+    </div>
+    <div>
+      <Button value="7" />
+      <Button value="8" />
+      <Button value="9" />
+      <Button value="X" />
+    </div>
+    <div>
+      <Button value="4" />
+      <Button value="5" />
+      <Button value="6" />
+      <Button value="-" />
+    </div>
+    <div>
+      <Button value="1" />
+      <Button value="2" />
+      <Button value="3" />
+      <Button value="+" />
+    </div>
+    <div>
+      <Button value="0" />
+      <Button value="." />
+      <Button value="=" />
+    </div>
+  </div>
```

Comment:
### [Optional]
I think we could make use of two loops to make the Button instead of repeating ourselves. let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

To do this; create an array of symbols and loop through the array using [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.